main.tf

```diff
@@ -15,8 +15,8 @@ locals {
       primary = var.ip_ranges.private_primary
       k8s = flatten([
         for k8s_ip_ranges in var.ip_ranges.private_k8s : [
-          { serial = index(var.ip_ranges.private_k8s, k8s_ip_ranges) + 1, cidr = k8s_ip_ranges.pods, name = format("private-k8spods%s-%s", "%s", var.name_suffix) },
-          { serial = index(var.ip_ranges.private_k8s, k8s_ip_ranges) + 1, cidr = k8s_ip_ranges.svcs, name = format("private-k8ssvcs%s-%s", "%s", var.name_suffix) }
+          { cidr = k8s_ip_ranges.pods, name = (k8s_ip_ranges.name != "") ?  format("%s-pods", k8s_ip_ranges.name) : format("private-k8spods%s-%s", (index(var.ip_ranges.private_k8s, k8s_ip_ranges) + 1) > 1 ? (index(var.ip_ranges.private_k8s, k8s_ip_ranges)) : "", var.name_suffix) },
+          { cidr = k8s_ip_ranges.svcs, name = (k8s_ip_ranges.name != "") ?  format("%s-svcs", k8s_ip_ranges.name) : format("private-k8ssvcs%s-%s", (index(var.ip_ranges.private_k8s, k8s_ip_ranges) + 1) > 1 ? (index(var.ip_ranges.private_k8s, k8s_ip_ranges)) : "", var.name_suffix) }
```

Comment:
Too long to read here. Can we break up the lines around the commas and brackets?